src/cljs/nr/deckbuilder.cljs

```diff
@@ -667,7 +681,7 @@
                                   (not valid) " invalid"))
                 :on-mouse-enter #(when (:setname card) (put! zoom-channel line))
                 :on-mouse-leave #(put! zoom-channel false)} title]
-        [card-influence-html format card modqty infaction allied]])
+        (card-influence-html format card modqty infaction allied)])
```

Comment:
it is important that functions that return hiccup (html) use square brackets. please see the reagent docs for why.

Reply:
ohhh okay i will resolve these. I was wondering what was happening as I was moving things around, moving things and putting them back

Reply:
@NoahTheDuke looking back at master, I think the reason I was confused is that the two instances of this function are formatted differently, one with `()` and the other with `[]`